src/kOS/Execution/CPU.cs

```diff
@@ -759,10 +759,7 @@ public void RemoveTrigger(int triggerFunctionPointer)
 
         public void StartWait(double waitTime)
         {
-            if (waitTime > 0)
-            {
-                timeWaitUntil = currentTime + waitTime;
-            }
+            timeWaitUntil = currentTime + waitTime;
             currentStatus = Status.Waiting;
```

Comment:
This was the cause of issue #688 .  The parameter waitTime comes directly from the "N" in the ```wait N.``` command, and to tell the CPU that this was the kind of WAIT that was a ```WAIT UNTIL FOO``` instead of a ```WAIT N```, the OpcodeAddTrigger(true) opcode would call cpu.StartWait(0).  The Cpu would then know it needed to rely on the execution of the triggers to see if the value had become true anywhere during the triggers.   But there was nothing preventing a ```wait N.``` from also passing in a value of N=0 and ending up calling cpu.StartWait(0) as well.

So the Cpu got confused and thought that ```wait 0.``` was really a ```wait until foo.``` but the compiler had never built the trigger for the expression that ```wait until```'s depend on, so it was always stuck on the default value of false.  Permanently.

But now that there are no trigger-based waits anymore, and the only kind of wait is a numeric seconds wait as far as the CPU cares (because the UNTIL logic exists out in user-code now not in the triggers), there's no problem with passing in a zero anymore.  Now it just means "wait until a tick in which the timer is later than right now", or in other words, the same thing as we had been using ```wait 0.0001